bots/sg_single_rule_delete.py

```diff
@@ -202,10 +203,10 @@ def run_action(boto_session, rule, entity, params):
                     text_output = text_output + 'Security Group ingress rule from port %s to port %s successfully added\n' % (
                         upper_port_number, upper_port_number_to)
 
-    if split == True:
-        if protocol_to_remove:
-            lower_port_number = -1
-            upper_port_number_to = 0
+    if split:
+        if '-1' == protocol_to_remove:  # -1 specify all protocols - bug fixed : '-1' cannot use as boolean
+            lower_port_number = 0
+            upper_port_number_to = -1
```

Comment:
I don't know if it doesn't break other things

Reply:
I have a new commit that fixes this issue